### PR TITLE
created constants

### PR DESCRIPTION
**Pull request by SamuelStone-beep**
its basically robotmap

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -18,4 +18,37 @@
...
+
+    public static final double CONTROLPANEL_MOTOR_ID = 14;
+    public static final double CONTROLPANEL_TRENCH_HARDSTOP_SOLENOID_ID = 4;
+
+    public static final double VISION_LIMELIGHT_ID = 31;
+
+
+
+
+
```
Comment (duffym22) [inline src/main/java/frc/robot/Constants.java]: WHAT ARE THESE EMPTY LINES

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -18,4 +18,37 @@
...
+    public static final double INDEXER_CONVEYER_ID = 15;
+    public static final double INDEXER_CELLGATE_ID = 3;
+
+    public static final double INTAKE_INTAKEMOTOR_ID = 16;
+    public static final double INTAKE_INTAKESOLENOID_ID = 0;
+
+    public static final double CONTROLPANEL_MOTOR_ID = 14;
+    public static final double CONTROLPANEL_TRENCH_HARDSTOP_SOLENOID_ID = 4;
+
+    public static final double VISION_LIMELIGHT_ID = 31;
```
Comment (duffym22) [inline src/main/java/frc/robot/Constants.java]: Change all instances of `double` to `int`

**Review by borkae — CHANGES_REQUESTED** [review]
Change doubles to ints

**Review by duffym22 — CHANGES_REQUESTED** [review]
- Remove empty lines
- Change all instances of `double` to `int`

**Review by borkae — APPROVED** [review]
Looks good.

**Review by duffym22 — APPROVED** [review]
Good changes

**Pull request merged**